docs/Contributors_Guide/continuous_integration.rst

```diff
@@ -119,12 +380,31 @@ Here is a list of the currently supported keywords and what they control:
 
 * **ci-skip-all**: Don't run anything - skip all automation jobs
 * **ci-skip-use-cases**: Don't run any use cases
+* **ci-skip-unit-tests**: Don't run the Pytest unit tests
 * **ci-run-all-cases**: Run all use cases
 * **ci-run-diff**: Obtain truth data and run diffing logic for
   use cases that are marked to run
 * **ci-run-all-diff**: Obtain truth data and run diffing logic for
   all use cases
-* **ci-only-docs**: Only run build documentation job - skip the rest
+
+.. _cg-ci-get-image:
+
+Create/Update METplus Docker Image
+----------------------------------
+
+This job calls the **docker_setup.sh** script (found in .github/jobs).
+This script builds a METplus Docker image and pushes it to DockerHub.
+The image is pulled instead of built in each test job to save execution time.
+The script attempts to pull the appropriate Docker image from DockerHub
+(dtcenter/metplus-dev:**<BRANCH_NAME>**) if it already exists so that unchanged
+components of the Docker image do not need to be rebuilt.
```

Comment:
Clarify a bit more. Are you saying, for example, that it first pull dtcenter/metplus-dev:develop to get many of the common layers, prior to recreating it from scratch? Is that correct? So the partial rebuild is faster than a complete rebuild?

Reply:
Yes, that is correct. However, the first step is to pull the MET Docker image, which is a likely piece that would change. If it does, the rest of the steps need to be rerun. If the MET Docker image has not changed in between runs, then I think this logic speeds things up quite a bit.